app/src/pages/inside/stepPage/modals/makeDecisionModal/makeDecisionModal.jsx

```diff
@@ -77,21 +75,25 @@ const MakeDecision = ({ data }) => {
     selectedItems: [],
     suggestedItems: [],
     startTime: Date.now(),
+    selectManualChoice: { issue: isBulkOperation ? { comment: '' } : itemData.issue },
+    suggestChoice: {},
+    historyChoice: {
+      ...historyItems.filter((item) => item.issue && item.id !== itemData.id)[0],
+    },
```

Comment:
You can skip spreading and wrapping item. Also better use `find` method here to avoid unnecessary iteration on items